### PR TITLE
Initial OZW Node Config Panel

### PR DESCRIPTION
**Pull request by cgarwood**
## Proposed change

Adds an initial page to the OZW config panel listing node configuration parameters and their current settings. Read-only for this PR. Future PRs will allow for updating these config parameters

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by cgarwood** [comment]
image

File: src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  @internalProperty() private _config?: OZWDeviceConfig[];
+
+  @internalProperty() private _not_found = false;
+
+  protected firstUpdated() {
+    if (!this.ozwInstance) {
+      navigate(this, "/config/ozw/dashboard", true);
+    } else if (!this.nodeId) {
+      navigate(this, `/config/ozw/network/${this.ozwInstance}/nodes`, true);
+    } else if (this.hass) {
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts:62]: `hass` should always be defined here.

File: src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+      );
+      this._config = await fetchOZWNodeConfig(
+        this.hass!,
+        this.ozwInstance,
+        this.nodeId
+      );
+    } catch (err) {
+      if (err.code === ERR_NOT_FOUND) {
+        this._not_found = true;
+        return;
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts:191]: For future proving, let's save the error code in `this._error` and render the error page with text based on that value.

File: src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+      this._metadata = await fetchOZWNodeMetadata(
+        this.hass!,
+        this.ozwInstance,
+        this.nodeId
+      );
+      this._config = await fetchOZWNodeConfig(
+        this.hass!,
+        this.ozwInstance,
+        this.nodeId
+      );
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts:187]: We now fetch this 1 for 1, we could do them all at once:
```
const nodeProm = fetchOZWNodeStatus(...);
const metaProm = fetchOZWNodeMetadata(...);
const configProm = fetchOZWNodeConfig(...);
[this._node, this._metadata, this._config] = await Promise.all([nodeProm, metaProm, configProm])`

File: src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+                      ${this._node.node_manufacturer_name}
+                      ${this._node.node_product_name} </b
+                    ><br />
+                    Node ID: ${this._node.node_id}<br />
+                    Query Stage: ${this._node.node_query_stage}
+                    ${this._metadata?.metadata.ProductManualURL
+                      ? html` <a
+                          href="${this._metadata.metadata.ProductManualURL}"
+                        >
+                          <p>Product Manual</p>
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts:118]: Lokalise

File: src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+                        </a>`
+                      : ``}
+                  </div>
+                  <div class="card-actions">
+                    <mwc-button @click=${this._refreshNodeClicked}>
+                      Refresh Node
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts:124]: Lokalise

File: src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+                    </mwc-button>
+                  </div>
+                </ha-card>
+
+                ${this._metadata?.metadata.WakeupHelp
+                  ? html`
+                      <ha-card class="content" header="Wake-up Instructions">
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts:131]: Lokalise

**Comment by cgarwood** [comment]
Review comments addressed and additional translations added 👍

**Review by bramkragten — APPROVED** [review]

**Pull request merged**